playground-configs/third-party/kube-prometheus-stack.yaml

```
@@ -73,3 +125,78 @@ spec:
+            annotations:
+              description: Job {{ $labels.job_name}} in environment {{ $labels.label_radix_env }} for application {{ $labels.label_radix_app}} has been in a non-ready state for more than 10 minutes.
+              summary: Job is not ready
+              consoleUrl: https://console.playground.radix.equinor.com/applications/{{ $labels.label_radix_app }}/envs/{{ $labels.label_radix_env }}/jobcomponent/{{ $labels.label_radix_component }}
+            for: 10m
+            expr: >-
+              sum by(label_radix_app, label_radix_env, label_radix_component, job_name, namespace)(
+                max by(namespace, pod) (kube_pod_status_phase{job="kube-state-metrics",namespace=~".*",phase=~"Pending|Unknown"}) 
+                * on(namespace, pod) group_left(owner_kind, job_name) topk by(namespace, job_name) (1, max by(namespace, pod, owner_kind, job_name) (label_replace(kube_pod_owner{owner_kind="Job"}, "job_name", "$1", "owner_name", "(.*)"))) 
+                * on(job_name, namespace) group_left(label_radix_app, label_radix_component) (kube_job_labels{label_radix_app!=""})
+                * on(namespace) group_left(label_radix_env) (kube_namespace_labels{label_radix_env!=""})
+              )>0
+          - alert: RadixAppJobFailed
+            labels:
+              severity: error
+              radixscope: env
+            annotations:
+              description: Job {{ $labels.job_name}} in environment {{ $labels.label_radix_env }} for application {{ $labels.label_radix_app}} failed.
+              summary: Job failed
+              consoleUrl: https://console.playground.radix.equinor.com/applications/{{ $labels.label_radix_app }}/envs/{{ $labels.label_radix_env }}/jobcomponent/{{ $labels.label_radix_component }}
+            expr: >-
+              max by(job_name, namespace) (kube_job_failed{job="kube-state-metrics",namespace=~".*"}) 
+              * on(job_name, namespace) group_left(label_radix_app, label_radix_component) kube_job_labels{job="kube-state-metrics",label_radix_job_type="job-scheduler"} 
+              * on(namespace) group_left(label_radix_env) (kube_namespace_labels{label_radix_env!=""}) > 0
+          - alert: RadixAppPipelineJobFailed
+            labels:
+              severity: error
+              radixscope: app
+            annotations:
+              description: Pipeline job {{ $labels.label_radix_job_name}} for application {{ $labels.label_radix_app}} failed.
+              summary: Pipeline job failed
+              consoleUrl: https://console.playground.radix.equinor.com/applications/{{ $labels.label_radix_app }}/jobs/view/{{ $labels.label_radix_job_name }}
+            expr: >-
+              max by(job_name, namespace) (kube_job_failed{job="kube-state-metrics",namespace=~".*"}) 
+              * on(job_name, namespace) group_left(label_radix_app, label_radix_job_name, label_radix_pipeline) kube_job_labels{job="kube-state-metrics",label_radix_job_type="job"} > 0                
```

not sure what you need this trailing whitespace for 😄
```suggestion
              * on(job_name, namespace) group_left(label_radix_app, label_radix_job_name, label_radix_pipeline) kube_job_labels{job="kube-state-metrics",label_radix_job_type="job"} > 0
```

---

fixed